test/test_helper.bats

```diff
@@ -198,6 +198,9 @@ load 'test_helper/common'
 }
 
 @test "wait_for_empty_mail_queue_in_container fails when timeout reached" {
+    # TODO investigate why these commands fail
+    skip 'disabled as it fails randomly: https://github.com/docker-mailserver/docker-mailserver/issues/2176'
```

Comment:
I would link to to 2177 (this PR) rather than the issue, as the PR seems to have more relevant information.